src/Net/TcpKeepAliveSettings.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Amqp
+{
+    /// <summary>
+    /// TCP Keep-Alive settings.
+    /// </summary>
+    public class TcpKeepAliveSettings
+    {
+        public ulong KeepAliveTime
```

Comment:
Can you also add some text for this and the other one? Right now warnings are treated as errors so the XML doc missing warning on these two is causing build error.